pkgs/tools/networking/zs-wait4host/default.nix

```diff
@@ -0,0 +1,42 @@
+{ bash
+, coreutils
+, fetchurl
+, fping
+, lib
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
Can we use stdenvNoCC here?

Reply:
afaik yes.

Reply:
This probably also applies to https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/text/zstxtns-utils/default.nix.